packages/frontend/src/index.css

```
@@ -142,6 +142,72 @@ button {
+  font-family: "JetBrains Mono", monospace;
+  font-weight: 800;
+  font-size: 14px;
+  letter-spacing: 0.18em;
+  border: 2px solid #fff;
+  cursor: pointer;
+  box-shadow: 0 0 32px rgba(255, 209, 102, 0.6);
+  animation: pressBlink 1s ease-in-out infinite;
+  z-index: 10;
+}
+
+@keyframes pressBlink {
```

Stylelint (17.9.0)

[error] Expected keyframe name "pressBlink" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)